Makefile

```diff
@@ -29,39 +23,17 @@ apt:
 		libnetcdf-dev \
 		libgdal-dev
 
-.PHONY: build-docs
-build-docs: venv
-	${PYTHON} setup.py build_sphinx
-
-.PHONY: clean
-clean:
-	rm -rf $(VENV_PATH)
+build-docs:
+	pipenv run python setup.py build_sphinx
 
-.PHONY: gdal-check
 gdal-check:
-	$(eval GDAL_REQ=$(shell cat requirements.txt | grep -i gdal | cut -c 7-))
-	$(eval GDAL_VER=$(shell gdal-config --version))
-
-	if ! [ "$(GDAL_REQ)" == "$(GDAL_VER)" ]; then \
+	if [ $(FOUND) = false ]; then \
```

Comment:
Curious: Why take this computation out of the make target? It looks like it is only used here.

Reply:
In order search the env var the way I wanted I had to pull it out of the target. The syntax works differently when items are in the target vs outside of it.